### PR TITLE
new(mp) New DRBD SSH Monitoring Procedure [FR]

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+id: applications-drbd-ssh
+title: DRBD
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:3]:
```suggestion
title: DRBD SSH
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+---
+
+## Vue d'ensemble
+
+DRBD (Distributed Replicated Block Device en anglais, ou périphérique en mode bloc répliqué et distribué en français) est une architecture de stockage distribuée pour GNU/Linux, 
+permettant la réplication de périphériques de bloc (disques, partitions, volumes logiques etc.) entre des serveurs. DRBD est un logiciel libre, mais un support existe. DRBD est composé d'un module noyau,
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:9]:
```suggestion
permettant la réplication de périphériques de bloc (disques, partitions, volumes logiques etc.) entre des serveurs.
DRBD est un logiciel libre, mais un support existe. DRBD est composé d'un module noyau, 
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources DRDB :
+
+```bash
+yum install centreon-plugin-Applications-Drbd-Ssh.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *DRBD* depuis la page "Configuration > Plugin packs > Manager"
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:61]:
```suggestion
2. Sur l'interface Web de Centreon, installer le Plugin-Pack *DRBD SSH* depuis la page "Configuration > Plugin packs > Manager"
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+yum install centreon-plugin-Applications-Drbd-Ssh.noarch
+```
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install ccentreon-pack-applications-drbd-ssh.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *DRBD* depuis la page "Configuration > Plugin packs > Manager"
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:77]:
```suggestion
3. Sur l'interface Web de Centreon, installer le Plugin-Pack *DRBD SSH* depuis la page "Configuration > Plugin packs > Manager"
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+| Mandatory   | Name            | Description                                                                                     |
+| :---------- | :-------------- | :---------------------------------------------------------------------------------------------- |
+| X           | SSHBACKEND      | Nom du backend: ```sshcli```                                                                    |
+| X           | SSHUSERNAME     | Par default, il utilise l'utilisateur en cours d'exécution ```centengine``` de votre collecteur |          
+|             | SSHPASSWORD     | Ne peut pas être utilisé avec le backend. Seulement avec la clé d'authentication                |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
+
+:warning: Avec ce backend, il est nécessaire d'effectuer une connexion manuelle entre l'utilisateur centreon-engine du Collecteur
+et l'utilisateur applicatif créé sur le serveur Linux. (Macro SSHUSERNAME).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:100]: Wrong traduction in the description fields

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+<!--plink backend-->
+
+| Mandatory   | Name            | Description                                                                                     |
+| :---------- | :-------------- | :---------------------------------------------------------------------------------------------- | 
+| X           | SSHBACKEND      | Nom du backend: ```plink```                                                                     |
+| X           | SSHUSERNAME     | Par default, il utilise l'utilisateur en cours d'exécution ```centengine``` de votre collecteur |
+|             | SSHPASSWORD     | Peut être utilisé. Si aucune valeur n'est définie, l'authentification par clé ssh est utilisée  |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:110]: Wrong trad

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+<!--libssh backend (par défaut)-->
+
+| Mandatory   | Name            | Description                                                                                     |
+| :---------- | :-------------- | :---------------------------------------------------------------------------------------------- |
+| X           | SSHBACKEND      | Nom du backend: ```libssh```                                                                    |          
+|             | SSHUSERNAME     | Par default, il utilise l'utilisateur en cours d'exécution ```centengine``` de votre collecteur |
+|             | SSHPASSWORD     | Peut être utilisé. Si aucune valeur n'est définie, l'authentification par clé ssh est utilisée  |
+|             | SSHPORT         | Par default: 22                                                                                 |
+|             | SSHEXTRAOPTIONS | Personnalisez-le avec le vôtre si nécessaire. E.g.: ```--ssh-priv-key=/user/.ssh/id_rsa```      |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:123]: Wrong trad'

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+```bash
+/usr/lib/centreon/plugins/centreon_drbd_ssh.pl \
+    --plugin=apps::drbd::local::plugin.pm \
+    --mode=resources \
+    --hostname=10.30.2.81 \
+	--ssh-username=centreon \
+	--ssh-password=centreon-password \
+	--ssh-backend=libssh \
+	--legacy-proc \
+	--verbose
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:142]:
```suggestion
/usr/lib/centreon/plugins/centreon_drbd_ssh.pl \
    --plugin=apps::drbd::local::plugin.pm \
    --mode=resources \
    --hostname=10.30.2.81 \
    --ssh-username=centreon \
    --ssh-password=centreon-password \
    --ssh-backend=libssh \
    --legacy-proc \
    --verbose
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+'6~0#peer.traffic.out.bitspersecond'=0b/s;;;0; '7#disk.data.read.bytespersecond'=0B/s;;;0; '7#disk.data.written.bytespersecond'=0B/s;;;0; '7~0#peer.traffic.in.bitspersecond'=0b/s;;;0; 
+'7~0#peer.traffic.out.bitspersecond'=0b/s;;;0; '8#disk.data.read.bytespersecond'=0B/s;;;0; '8#disk.data.written.bytespersecond'=0B/s;;;0; '8~0#peer.traffic.in.bitspersecond'=0b/s;;;0; 
+'8~0#peer.traffic.out.bitspersecond'=0b/s;;;0;	
+```
+
+La commande ci-dessus contrôle les resources l'application DRBD (```--mode=resources```). 
+Il y a pour adresse 10.30.2.81 (```--hostname=10.30.2.81```) comme Backend SSH  (```--ssh-backend='libssh'```) 
+avec les centreon comme username _centreon_ (```--ssh-username=centreon```) et comme mot de passe _centreon-password_ (```--ssh-password='centreon-password'```). 
+
+L'option _legacy-proc_ (```--legacy-proc```) permet de pouvoir utiliser l'ancien fichier proc (```/proc/drbd```). Ce qui permet d'utiliser une version les versions antérieurs à la version 9.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:161]:
```suggestion
L'option _legacy-proc_ (```--legacy-proc```) permet de pouvoir utiliser l'ancien fichier proc (```/proc/drbd```). Ce qui permet d'utiliser une version les versions antérieures à la version 9 de DRBD.
```

File: fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```bash
+/usr/lib/centreon/plugins/centreon_drbd_ssh.pl --plugin=apps::drbd::local::plugin.pm \
+--mode=resources --help
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-drbd-ssh.md:167]:
```suggestion
  --mode=resources --help
```

**Review by pkriko — CHANGES_REQUESTED** [review]

**Review by Sims24 — CHANGES_REQUESTED** [review]
Please see my review here: https://github.com/centreon/centreon-documentation/pull/363

**Review by Sims24 — APPROVED** [review]

**Review by pkriko — APPROVED** [review]

**Pull request merged**